app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/index.tsx

```diff
@@ -249,17 +271,19 @@ const SelectColumnsView: React.FC<ISelectColumnsProps> = (props) => {
           label={`${props.tableInfo.table} - Mappings, assessments and transformations`}
         />
         <ActionButtons>
+          {/* 
+            Will need to add this back for the 6.6 version
```

Comment:
Can you file a bug for 6.6 to make sure we turn this back on?

Reply:
probably should've linked it though :)